.github/workflows/frontent-ci-pipeline.yml

```
@@ -0,0 +1,36 @@
+name: "Frontend test - action"
```

```suggestion
name: "Frontend Lint & Compile check"
```
Just to avoid confusion that this doesn't run any actual npm tests. Could we also update the filename to something like `frontend-lint-and-compile-check.yaml` ?